faq.yml

```diff
@@ -0,0 +1,94 @@
+- 
```

Comment:
imo this way of formatting arrays is strange (- with empty new line) We don't use it in content.yml so i would suggest not using it here either.